README.md

```diff
@@ -78,4 +78,36 @@ Example for basic operations.
 
         }
 
-```
\ No newline at end of file
+```
+
+## Development
```

Comment:
Could you explicitly add that this section is for uplink-java maintainers?

Reply:
sure, done.